services/graphql-server/src/graphql/resolvers/platform/content.js

```diff
@@ -918,8 +918,18 @@ module.exports = {
         excludeContentTypes,
       });
 
-      customAttributes.forEach(({ key, value }) => {
-        query.$and.push({ [`customAttributes.${key}`]: value });
+      customAttributes.forEach(({ key, value, exists }) => {
+        const valueOrExists = Boolean(value || typeof exists !== 'undefined');
```

Comment:
Technically, this should check that `typeof exists === 'boolean'`. We want to error if there is no value and exists is nullish.